example/src/screens/Room.tsx

```diff
@@ -1,181 +1,187 @@
+            flex: 1,
+            maxHeight: width - 2 * PADDING_BETWEEN_PARTICIPANTS,
+            maxWidth: width - 2 * PADDING_BETWEEN_PARTICIPANTS,
+          },
+    ];
+  };
+
   return (
-    <View style={styles.flex}>
-      <View style={styles.flex}>
-        {!!focusedTrack && !!focusedParticipant && (
-          <View
-            style={[
-              styles.focusedParticipantContainer,
-              isFocusedParticipantSpeaking ? styles.vadSpeech : {},
-            ]}
-          >
-            <Membrane.VideoRendererView
-              trackId={focusedTrack.id}
-              style={styles.focusedParticipant}
-              videoLayout={Membrane.VideoLayout.FIT}
-            />
-            <Text style={styles.displayName}>
-              {focusedParticipant.metadata.displayName}
-            </Text>
-            {!!areSettingsOpen && (
-              <View style={styles.settingsWrapper}>
-                <Settings
-                  participant={focusedParticipant}
-                  track={focusedTrack}
-                />
-              </View>
-            )}
-            <View style={styles.disabledIconsContainer}>
-              {!focusedParticipant.tracks.find((t) => t.type === 'Audio')
-                ?.metadata.active && <Icon name="Microphone-off" size={24} />}
-              {!focusedTrack.metadata.active && (
-                <Icon name="Cam-disabled" size={24} />
+    <BackgroundAnimation>
+      <View style={styles.container}>
+        <View style={styles.header}>
+          <View style={styles.headerTitle}>
+            <Typo variant="h4">{roomName}</Typo>
+          </View>
+          <View style={styles.headerIcon}>
+            <Pressable onPress={switchCamera}>
+              <Icon
+                name="Cam-switch"
+                size={24}
+                color={BrandColors.darkBlue100}
+              />
+            </Pressable>
+          </View>
+        </View>
+
+        <View style={styles.flex}>
+          <View style={styles.participantsContainer}>
+            <View
+              style={[
+                styles.inner,
+                participants.length > FLEX_BRAKPOINT
+                  ? styles.row
+                  : styles.column,
+              ]}
+            >
+              {participants
+                .slice(
+                  0,
+                  participants.length > MAX_NUM_OF_USERS_ON_THE_SCREEN
+                    ? MAX_NUM_OF_USERS_ON_THE_SCREEN - 1
+                    : MAX_NUM_OF_USERS_ON_THE_SCREEN
+                )
+                .map((p) => (
+                  <RoomParticipant
+                    key={p.id}
+                    participant={p}
+                    pStyle={getStylesForParticipants(participants)}
+                  />
+                ))}
+
+              {participants.length > MAX_NUM_OF_USERS_ON_THE_SCREEN && (
+                <View style={getStylesForParticipants(participants)}>
+                  <Typo variant="label">Others</Typo>
```

Comment:
it appears to be broken / not finished and there some new margins?
![Screenshot_20230328_161719](https://user-images.githubusercontent.com/6280457/228275535-be88e424-3d5b-4d8c-9559-462b47b8c1de.png)

Reply:
I said in the PR description that I planned to submit it it a different PR since making it look/work properly won't be a small thing and this PR is already big. Wdyt?

Reply:
ok, do it in the separate PR 👍 
but please look into the margin issue

Reply:
Fixed margin.